src/main/java/org/apache/sysml/api/jmlc/Connection.java

```diff
@@ -63,6 +54,15 @@
 import org.apache.sysml.runtime.util.DataConverter;
 import org.apache.wink.json4j.JSONObject;
 
+import java.io.BufferedReader;
+import java.io.Closeable;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.HashMap;
+import java.util.Map;
```

Comment:
Please, fix the ordering of imports - why did you replace the existing sorted imports with unsorted lists - there are multiple files with similar problems.